### PR TITLE
Added some error info for OPA policy fail

### PR DESCRIPTION
**Pull request by gyliu513**

### CODE REVIEW

**Review by srenatus — APPROVED** [review]
👍 this seems useful!

**Review by patrick-east — COMMENTED** [review]
I like the change! It's a nice improvement for the tutorial. Just a couple of nit's and we should be good to go.

**Comment by patrick-east** [comment]
@gyliu513 if you have a chance please update with the suggested changes so we can get this merged! 😄

**Comment by patrick-east** [comment]
Going to close this due to inactivity. Please re-open as needed 😄

File: docs/content/kubernetes-tutorial.md [hunk 1]
```diff
@@ -443,6 +443,12 @@ kubectl create -f ingress-bad.yaml -n qa
 
 The second Ingress is rejected because its hostname does not match the whitelist in the `qa` namespace.
 
+It will report error as follows:
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply docs/content/kubernetes-tutorial.md:446]: Done

File: docs/content/kubernetes-tutorial.md [hunk 2]
```diff
@@ -500,6 +506,12 @@ kubectl create -f staging-namespace.yaml
 kubectl create -f ingress-ok.yaml -n staging
 ```
 
+The above command will report error as follows:
```
[the remark this answers is not in the document]
Comment (gyliu513) [reply docs/content/kubernetes-tutorial.md:509]: Done

**Comment by gyliu513** [comment]
@patrick-east can you help re-open and I will update, thanks!

**Comment by patrick-east** [comment]
Hmm It might require a new one to be opened, I'm seeing the button disabled with this message:
image

**Comment by gyliu513** [comment]
Thanks @patrick-east , created a new PR at https://github.com/open-policy-agent/opa/pull/2907

**Pull request closed without merging**